mavlink/modules/flight_controller.py

```diff
@@ -4,8 +4,9 @@
 
 import time
 
-from .. import dronekit
+from pymavlink import mavutil
```

Comment:
why does our version of dronekit not include mavutil?

Reply:
I'm pretty sure dronekit never included mavutil. It's just that dronekit depended upon mavutil, and it somehow worked?